modules/widgets/simple-payments.php

```diff
@@ -388,8 +388,8 @@ private function record_event( $stat_name, $event_action, $event_properties = ar
 
 			// `bumps_stats_extra` only exists on .com
 			if ( function_exists( 'bump_stats_extras' ) ) {
-				require_lib( 'tracks/client' );
-				tracks_record_event( $current_user, 'simple_payments_button_' . $event_action, $event_properties );
+				jetpack_require_lib( 'tracks/client' );
+				jetpack_tracks_record_event( $current_user, 'simple_payments_button_' . $event_action, $event_properties );
```

Comment:
this block of code is for WordPress.com, where `jetpack_*` functions are not available.

Reply:
Updated 👍